WalletWasabi/Extensions/LinqExtensions.cs

```diff
@@ -208,4 +208,13 @@ public static IEnumerable<T> TakeUntil<T>(this IEnumerable<T> list, Func<T, bool
 			}
 		}
 	}
+	
+	public static IEnumerable<T> ForEach<T>(this IEnumerable<T> source, Action<T> action)
```

Comment:
No please. `ForEach` is a bad idea and for that reason it was never included in dotnet. I am okay with it but only in for testing so, please move it to testing project.